articles/libraries/lock/v10/auth0js.md

```diff
@@ -1,75 +1,77 @@
 ---
 section: libraries
-description: How to use Lock V10 with auth0.js
+description: How to use Lock v10 with auth0.js
 ---
 # Using Lock With auth0.js
 
-By nature, Lock and the Auth0.js SDK are very different things. Lock provides a UI that is customizable, to an extent, with behavior that is customizable, to an extent. It is an easily deployed, easily used interface for Auth0 authentication in custom applications. For simple uses, Lock is all that is necessary, sometimes. However, if more customization is requiredin an application's signup, authentication, and user management process, then functionality from the SDK can be used alongside Lock to meet those needs.
+By nature, Lock and the Auth0.js SDK are different things. Lock provides a UI that is customizable (to an extent) with behavior that is customizable (to an extent). It is an easily deployed, easily used interface for Auth0 authentication in custom applications. It is also easily used within the Auth0 [Hosted Login Page](/hosted-pages/login).
 
-## Changes with newer versions
-
-If you try to use auth0.js along with Lock 10, you will not be able to call `getClient()`. Instead, you should include its dependency and instantiate the object.
-
-In the past, a version of auth0.js was included automatically with Lock; at this point, you must include auth0.js yourself, allowing you to choose the supported version of auth0.js that best suits the needs of your application - version 7, or version 8. Note that the two versions have different CDN paths, as well as slightly different instantiations.
+For simple uses, Lock is all that is necessary. However, while using Lock, if more customization is required in an application than Lock allows, functionality from the Auth0.js SDK can be used alongside Lock to meet those needs. An example might be using Lock to handle signups and logins, while using auth0.js to [manage users](/libraries/auth0js#user-management) (read and update user metadata, link user accounts together, and similar tasks).
 
 ## Using auth0.js v8
```

Comment:
This header is no longer needed since the paragraph includes both v7 and v8.

Reply:
Also, the first snippet is `Using Auth0.js v8/Using Auth0.js v7`, while the second one is `Auth0.js v8/Auth0.js v7`. Use one consistently.